FSI/FSI.C

```diff
@@ -63,10 +80,21 @@ void preciceAdapter::FSI::FluidStructureInteraction::addWriters(std::string data
         (
             dataName,
             // TODO: Hard-coded number of locations! Fix!!!
-            new Displacement(mesh_, runTime_, 68) /* TODO: Add any other arguments here */
+            // new Displacement(mesh_, runTime_, 162) /* TODO: Add any other arguments here */
+            new Displacement(mesh_, namePointDisplacement_) /* TODO: Add any other arguments here */
         );
         DEBUG(adapterInfo("Added writer: Displacement."));
     }
+    // else if (dataName.find("Velocity") == 0)
```

Comment:
This is the case for the velocityLaplacian. For reasons I explained you earlier I am first looking at displacementLaplacian. In this case, the velocity would need to be updated on the boundary, that's why I already included these commented lines. 

I think it is good to make the adapter suitable for multiple mesh deformation methods, and as I promised I will present you with a list of mesh deformation methods shortly.

Reply:
Conclusion from our meeting: let's indeed keep the displacementLaplacian and try to avoid exchanging velocities, as this is not currently an option in the CalculiX adapter.

The previous way (meshU = (displacement - displacementOld)/couplingTimeStepSize ) introduces an error and destroys any higher-order scheme used by the solver. However, we need somewhere to start from.

Other ideas:

* Define two meshes for the displacements, one on the face nodes and one on the face centers and use the displacements mapped on the face nodes for computing the velocity.
* Still define one mesh for the displacements, but on the face centers instead of the face nodes. Let the adapter do the interpolation to points.

Review conclusion: we don't need these lines, please remove them and we are going to work on this issue separately.

Reply:
Done!